apps/skilavottord/ws/src/app/modules/recyclingRequest/recyclingRequest.resolver.ts

```diff
@@ -64,17 +101,15 @@ export class RecyclingRequestResolver {
 
   @Mutation(() => RecyclingRequestResponse)
   async createSkilavottordRecyclingRequest(
+    @CurrentUser() user: User,
     @Args({ name: 'requestType', type: () => RecyclingRequestTypes })
     requestType: RecyclingRequestTypes,
```

Comment:
Missing check for this ... should citizen be able to set any kind of recyclingRequestType?